src/ModulusRemainder.cpp

```diff
@@ -376,65 +376,64 @@ int64_t lcm(int64_t a, int64_t b) {
     b /= gcd(a, b);
 
     // Then multiply
-    if (mul_would_overflow(64, a, b)) {
-        return 0;
-    } else {
-        return a * b;
-    }
+    int64_t result;
+    mul_with_overflow(64, a, b, &result);
```

Comment:
Why do we ignore the return value here? Worth adding a comment if deliberate.

Reply:
Done